src/apis/entries_api.rs

```diff
@@ -74,25 +84,40 @@ pub async fn create_log_entry(configuration: &configuration::Configuration, prop
     let local_var_content = local_var_resp.text().await?;
 
     if !local_var_status.is_client_error() && !local_var_status.is_server_error() {
-        serde_json::from_str(&local_var_content).map_err(Error::from)
+        // Format the returned response such that it can be read into a struct
+        let uuid: &str = &local_var_content[1..67];
+        let rest: &str = &local_var_content[69..local_var_content.len() - 2];
+        let sum = "{\"uuid\": ".to_string() + uuid + "," + rest;
+        serde_json::from_str::<LogEntry>(&sum).map_err(Error::from)
     } else {
-        let local_var_entity: Option<CreateLogEntryError> = serde_json::from_str(&local_var_content).ok();
-        let local_var_error = ResponseContent { status: local_var_status, content: local_var_content, entity: local_var_entity };
+        let local_var_entity: Option<CreateLogEntryError> =
+            serde_json::from_str(&local_var_content).ok();
+        let local_var_error = ResponseContent {
+            status: local_var_status,
+            content: local_var_content,
+            entity: local_var_entity,
+        };
         Err(Error::ResponseError(local_var_error))
     }
 }
 
-pub async fn get_log_entry_by_index(configuration: &configuration::Configuration, log_index: i32) -> Result<::std::collections::HashMap<String, serde_json::Value>, Error<GetLogEntryByIndexError>> {
+// Change the return value of the function to LogEntry from ::std::collections::HashMap<String, serde_json::Value>
+pub async fn get_log_entry_by_index(
+    configuration: &configuration::Configuration,
+    log_index: i32,
+) -> Result<LogEntry, Error<GetLogEntryByIndexError>> {
     let local_var_configuration = configuration;
```

Comment:
Why do you need to do that? Both variables will be pointing to the same area or memory 🤔